JS.md

```diff
@@ -608,6 +619,30 @@ accessor function performs the following steps:
 If `this` is not a `WebAssembly.Memory`, a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror)
 is thrown. Otherwise return `M.[[BufferObject]]`.
 
+### `WebAssembly.Memory.prototype.byteLength`
+
+[⚜](#Post-MVP-JavaScript-API)
+
+This is an accessor property whose [[Set]] is Undefined and whose [[Get]]
+accessor function performs the following steps:
+
+Let `T` be the `this` value. If `T` is not a `WebAssembly.Memory`, a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror)
+is thrown.
+
+Return `T.[[BufferObject]].[[ArrayBufferByteLength]]`.
```

Comment:
What is the motivation for this feature, when a user can do `memory.buffer.byteLength`? Is it more optimizable if you never access the ArrayBuffer, or something like that?

Reply:
Yeah creating the array buffer is expensive-ish. We can optimize it away, but that's being extra silly IMO.